inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/starrocks/StarRocksDataNodeOperator.java

```diff
@@ -88,7 +89,7 @@ protected void setTargetEntity(DataNodeRequest request, DataNodeEntity targetEnt
 
     @Override
     public Boolean testConnection(DataNodeRequest request) {
-        String jdbcUrl = request.getUrl();
+        String jdbcUrl = MySQLDataNodeDTO.convertToJdbcurl(request.getUrl());
```

Comment:
Better to use the convert method in StarRocksDTO

Reply:
StarRocks jdbc url rules are the same as mysql, rewriting a copy of the same would make the code duplicate.

Reply:
Rather than duplicating code, we recommend using classes associated with it within the class, rather than using non-utility classes from elsewhere.